### ISSUE TITLE
Issue with license import again

### ISSUE THREAD

**Issue by gintuoklis** [HodorNV/ALOps#104]
Hey Guys, 

This license import task is tough cookie :)

I get my license by downloading to the working folder:

```
      - task: DownloadPackage@1
        displayName: 'Download Package license file'
        inputs:
          packageType: upack
          feed: '9ce20875-af67-4251-abf2-6decb112898c/3c947a61-d29d-4d2a-8873-aef36c0d5f8e'
          definition: 'customer_license'
          version: '*'
          downloadPath: '$(System.DefaultWorkingDirectory)/license'
          files: fin.flf 
```

I can see license gets downloaded succesfully

![image](https://user-images.githubusercontent.com/16342128/78376836-281eab00-75cf-11ea-94a7-be3686e0a8f3.png)

but if i try to timport it, ALOpsLicenseImport cannot find the file:
I believe i tried all possible variations of license_path property
```
      - task: Hodor.hodor-alops.alopslicenseimport.ALOpsLicenseImport@1
        displayName: 'ALOps License Import'
        inputs:
          nav_serverinstance: buildTest
          license_path: '.\license\fin.flf'
          print_license_info: true
```
It always give me the same error message
```
##[error]The path cannot be processed because it resolved to more than one file; only one file at a time can be processed.
````

I already had similiar case with another ALOpsTask type. See reported issue #62 
Any clues, what I'm doing wrong?

**Comment by AdminHodor** [comment]
Dear @gintuoklis ,

The license import should be the easy one :-) 
Could you please provide the pipeline output for the failing step, hard to say what is going on without the output.

In the meantime we are adding extra examples to our documentation that show how the use the LicenseImport in different scenarios:
 - From Download Url
 - From Repository
 - From Docker Host

Kind regards,

**Comment by gintuoklis** [comment]
here's the full output:

```
Starting: ALOps License Import
==============================================================================
Task         : ALOps License Import
Description  : ALOps License Import
Version      : 1.428.1179
Author       : Hodor
Help         : Import Business Central license (.flf).
==============================================================================
*** Validate configuration
*** Importing required PS-Functions
*** Import license
##[error]The path cannot be processed because it resolved to more than one file; only one file at a time can be processed.
Finishing: ALOps License Import

```

**Comment by AdminHodor** [comment]
Dear @gintuoklis ,

I see, this does not seem a problem with the license file but rather with the loading of the NAV Cmdlets. 

Do you have multiple versions of NAV/BC installed on this machine? The script assumes there is only one version of NAV/BC installed. If this is the case, I'll have to look in dynamically resolving the correct Cmdlets for the selected ServiceTier. 

Kind regards,

**Comment by gintuoklis** [comment]
Yes, i have multiple NAV version on that server. 
I think i can help u with that. This is the code I use in my scripts to find the right folder of NAV instance.

```
$NAVServerExecutable = Get-WmiObject win32_service | ?{$_.Name -like "*`$$instanceName"} | select @{Name="Path"; Expression={$_.PathName.split('"')[1]}}
$NAVServerFolder = Split-Path $NAVServerExecutable.Path
Import-Module "$NAVServerFolder\NavAdminTool.ps1"
```

**Comment by AdminHodor** [comment]
Dear @gintuoklis ,

Glad we found the issue. I'll try to get a hotfix out asap, will keep you posted.

Kind regards,

**Comment by AdminHodor** [comment]
Dear @gintuoklis ,

We made a hotfix for the Multiple ServiceTier installations, which should address this issue.
Hotfix is included in release v1.428.1187.

Kind regards,

**Comment by gintuoklis** [comment]
Thank You for the swift support, it solved the issue
